core/app/models/spree/return_authorization.rb

```diff
@@ -20,6 +21,8 @@ class ReturnAuthorization < Spree::Base
     validate :must_have_shipped_units, on: :create
     validate :no_previously_exchanged_inventory_units, on: :create
 
+    validates :recipient, presence: true, if: :override_recipient
+    attr_accessor :override_recipient
```

Comment:
Just curious if this volatile attribute is necessary, could we determine that recipient was being overridden by the presence of a `recipient`?